woocommerce/migrator-cli
# Add Subscriptions Migrator

This PR adds a way to migrate Subscriptions


### Test instructions
- Checkout to this PR's branch
- Create two orders with the same product
- Update the orders creation date via the /wp-admin order edit page to `2023-01-07` and `2023-01-08`
- Add a `_order_number` meta and set it to `11664` in the oldest order
- Add a `_order_number` meta and set it to `11665` in the newest order
- Download these files
[orders.json](https://github.com/woocommerce/migrator-cli/files/13044179/orders.json)
[subscriptions.json](https://github.com/woocommerce/migrator-cli/files/13044180/subscriptions.json)
- Run `wp migrator skio_subscriptions --subscriptions_export_file=<file_path>  --orders_export_file=<file_path>`
- Make sure the subscription was created and uses the shipping and billing data from the newest order
- Change the shipping and billing address in the newest order
- Run `wp migrator skio_subscriptions --subscriptions_export_file=<file_path>  --orders_export_file=<file_path>` again
- Make sure the subscription got updated
- Change the status from `ACTIVE` to `CANCELLED` in the subscription.json file
- Run `wp migrator skio_subscriptions --subscriptions_export_file=<file_path>  --orders_export_file=<file_path>` again
- Make sure the subscription is now cancelled
- Run any other tests you think need to run
